packages/app-store/slack/README.md

```diff
@@ -0,0 +1 @@
+// TODO - install app, create bot in Slack (correct redirect URIs and scopes), add client_id and client_secret.
```

Comment:
@rkreddy99 
would be nice to have instructions on installing the app and setting up a bot in Slack UI

Reply:
Yeah this one is actually a requirement. @rkreddy99